spec/parallel_tests/test/runtime_logger_spec.rb

```diff
@@ -28,7 +28,7 @@ def test_foo
 
           class Bar#{i} < Test::Unit::TestCase
             def test_foo
-              sleep 0.25
+              sleep 0.2
```

Comment:
It didn't help, didn't break anything either. I'm trying new values now, if that doesn't work then I'll just skip this test on jruby.